keyboards/zinc/keymaps/toshi0383/keymap.c

```diff
@@ -0,0 +1,163 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+extern uint8_t is_master;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+    _QWERTY = 0,
+    _LOWER,
+    _RAISE,
+    _ADJUST,
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+
+enum macro_keycodes {
+  KC_SAMPLEMACRO,
+};
+
+
+//Macros
+#define M_SAMPLE M(KC_SAMPLEMACRO)
+
```

Comment:
You don't need Lines 25-32 at all. They should be removed.

Reply:
0299bb0c39851c4b834b4a3ad31fde9cb207bed6